.flake8

```diff
@@ -13,6 +13,8 @@ ignore =
     EXE001,
     # these ignores are from flake8-bugbear; please fix!
     B007,B008,B017,B019,B023,B028,B903,B904,B905,B906,B907
+    # Coding magic comment not found - do we really want to add encoding to every file?
+    C101,
```

Comment:
Please add codepage comments to those files instead (or strip the existing ones)
```suggestion
```